frontend/src/metabase/query_builder/actions/core/initializeQB.ts

```diff
@@ -403,11 +403,17 @@ async function handleQBInit(
   }
 }
 
-export const initializeQB =
+export const initializeQBRaw =
```

Comment:
1. Let me know if the name is weird
2. I moved this here so we can use the error state in our component

Reply:
I'll stick with `raw` right now since it seems to match a few other conventions in our code, but I'll add the comment 😄